api/v1beta1/common_types.go

```diff
@@ -84,6 +84,12 @@ const (
 	// on the reconciled object.
 	PausedAnnotation = "cluster.x-k8s.io/paused"
 
+	// MoveAnnotation is an annotation that will be applied to Cluster API
+	// object to mark that object is going to be moved to another management cluster.
+	//
+	// It will also help any validation webhook to take decision based on it.
+	MoveAnnotation = "cluster.x-k8s.io/move"
```

Comment:
I think this should be part of the `clusterctl` API package `v1alpha3/annotations.go` instead of the CAPI core API package. It would be good to name this carefully to avoid confusion with the labels used to manually move objects - https://github.com/kubernetes-sigs/cluster-api/blob/f2ba999757aa76b446943a9ffea14ecf8c7523e8/cmd/clusterctl/api/v1alpha3/labels.go#L35

Reply:
Hi @killianmuldoon 
Thanks for the nice finding. I have moved the constant accordingly and tried to find out a better name for the variable.